server/metadata/sqlitegen/query_program_test.go

```
@@ -21,6 +21,56 @@ type sqliteInstruction struct {
+
+func requireQueryUsesAnyTableIndex(t *testing.T, db *sql.DB, query string, tableName string, args ...any) {
+	t.Helper()
+	instructions := queryProgram(t, db, query, args...)
+	rows, err := db.Query(
+		`SELECT rootpage FROM sqlite_schema WHERE type = 'index' AND tbl_name = ?`,
+		tableName,
+	)
+	if err != nil {
+		t.Fatalf("resolve indexes for table %q: %v", tableName, err)
+	}
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)